index.html

```diff
@@ -0,0 +1,213 @@
+        </article>
+
+        <a href="#" class="button section__button">View all</a>
+      </section>
+
+      <section class="section main__section" id="howtobuy">
+        <div class="about">
+          <h2 class="title title--secondary about__title">
+            Why buy direct from Bose
+          </h2>
+          <div class="benefits about__benefits">
+            <p class="benefits__highlight">
+              Free 2-day shipping and returns
+            </p>
+
+            <p class="benefits__highlight">
+              90-day risk-free trial
+            </p>
+
+            <p class="benefits__highlight">
+              World class customer service
+            </p>
+
+            <p class="benefits__highlight">
+              My Bose account management
+            </p>
+
+            <p class="benefits__paragraph">
+              A great product is more than what’s in the box. It’s a promise of premium performance, world-class
+              support,
+              and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with
+              confidence on Bose.com.
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section main__section" id="contacts">
+        <h2 class="title title--secondary main__title">
+          Contact us
+        </h2>
+        <div class="contact">
+          <form class="form contact__form">
+            <div class="form__item">
+              <label for="name" class="form__label">Name</label>
+              <input id="name" class="form__input" type="text" placeholder="Name" required>
+            </div>
+
+            <div class="form__item">
+              <label for="email" class="form__label">Email</label>
+              <input id="email" class="form__input" type="email" placeholder="Email" required>
+            </div>
+
+            <div class="form__item form__item--message">
+              <label for="message" class="form__label">Message</label>
+              <textarea id="message" class="form__textarea" placeholder="Message" required></textarea>
+            </div>
+            <button class="button button--primary">Send</button><br><br>
+          </form>
+
+          <address class="address contact__address">
+            <p class="address__label">Phone</p>
+            <p class="address__content">+1 234 5555-55-55</p>
+
+            <p class="address__label">Email</p>
+            <p class="address__content">hello@miami.com</p>
+
+            <p class="address__label">Address</p>
+            <p class="address__content">
+              <a href="https://goo.gl/maps/U9Uwjf3qTQoKgCJs8" target="_blank">
+              first ave.<br>
+              suite 700<br>
+              Minneapolis, MN 55401
+              </a>
+            </p>
+          </address>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer"></footer>
+  </body>
+</html>
\ No newline at end of file
```

Comment:
Files should end with a newline to ensure better compatibility across different systems and tools.